docs/development.md

```diff
@@ -64,58 +51,57 @@ $ pnpm install
 
-You can just kill the terminal (`CTRL + C` usually) when you want to stop the server.
+> ### For Windows:
+>
+> You may encounter various errors involving file permissions if using Windows-based tools alongside WSL2. It is recommend that you fix these issues by either granting the correct file permissions to any that Windows may have modified, or by redownloading the repository from github. 
 
-If for some reason development resources aren't shutting down correctly, you can run the following:
+Once everything has started, you can connect to `http://www.wikijump.localhost/`. Changes you make to the codebase should automatically be applied to the containers, as your machine's filesystem has been "bound" to the containers' filesystem. This is one-way, so a container can't modify your filesystem. Adding new dependencies, however, will require a rebuild.
 
-```sh
-$ pnpm dev clean
-```
+You can just kill the terminal (`CTRL + C` usually) when you want to stop the server.
 
 If you want to entirely _reset_ the containers, as their data is otherwise persistent even across restarts, you can run the following:
 
 ```sh
-$ pnpm compose down
+$ docker-compose -p wikijump -f docker-compose.yaml -f docker-compose.dev.yaml down
 ```
 
-You can also bring containers up without the JS/TS development mode.
+It's useful to keep track of existing Docker images and containers, and destroy them when you no longer need them, so you don't waste space rebuilding the same image over and over. If you are using Docker Desktop, you can manage containers and images from the GUI. Otherwise, using the command line:
+
 ```
-$ pnpm dev serve
+$ docker container ls  # List containers
+$ docker rm [ID]       # Destroy the container with this ID
+$ docker images        # List images
+$ docker rmi [ID]      # Remove the image with this ID
 ```
 
-## Using `sudo`
-
-If your environment requires docker commands be run using `sudo`, the normal `pnpm dev` and `pnpm compose` commands will fail. You can use the `sudo` variants of these commands, `pnpm dev sudo` and `pnpm compose-sudo`.
 
 ## Entering the container
 
-If you want to enter the container to make temporary changes, you can do so by entering it with a CLI. From Docker Desktop, after running the containers, find the Wikijump app and within it the container you wish to enter, then click the 'CLI' button. Or from the command line:
+If you want to enter a container to make temporary changes, you can do so by entering it with a CLI. From Docker Desktop, after running the containers, find the Wikijump app and within it the container you wish to enter, then click the 'CLI' button. Or from the command line:
 
 ```
-$ docker exec -it [name of container] sh
+$ docker exec -it [container id] sh
 ```
 
-...where `[name of container]` is the name of the container from `docker container ls`.
+...where `[container id]` is the ID of the corresponding container from `docker ps`.
```

Comment:
Doesn't `docker container ls` also work?

Reply:
Yes, though `docker ps` is shorter. They are functionally identical, though I figured having it be the shorter command would make it easier to remember. I can revert this change if you think `container ls` is more recognizable though.